R/tools.R

```diff
@@ -32,3 +32,95 @@ call_tool <- function(fun, arguments) {
     }
   )
 }
+
+get_help_text <- function(topic, package = NULL) {
+  file_contents <- NULL
+  fake_pager <- function(files, header, title, delete.file) {
+    if (delete.file) {
+      on.exit(unlink(files))
+    }
+
+    for (file in files) {
+      file_contents <<- c(
+        file_contents,
+        readLines(files, warn = FALSE),
+        "\n"
+      )
+    }
+  }
+
+  op <- options(help_type = "text", pager = fake_pager)
+  on.exit(options(op), add = TRUE)
```

Comment:
There are various slightly more sophisticated techniques we could use here if we wanted to restrict the documentation to just the parameters of interest. But that's something we can easily do later.